lib/mayaUsd/fileio/translators/translatorMayaReference.cpp

```diff
@@ -138,6 +142,20 @@ const MObject getMessageAttr()
     return messageAttr;
 }
 
+const TfToken maya_associatedReferenceNode("maya_associatedReferenceNode");
+const bool    isMayaReference(const UsdPrim& prim)
+{
+    if (TfGetenvBool("MAYAUSD_ENABLE_MAYA_REFERENCE_NEW_BEHAVIOUR", false)) {
+        return true; // Force new behaviour for all Maya Reference prims.
+    } else {
+        const TfToken MayaReference("MayaReference");
+        return prim.GetTypeName()
+            == MayaReference; // Only MayaReference prims are using the new behaviour.
+    }
+}
+
+const TfToken MayaReferenceAttribute("MayaReferenceAttribute");
```

Comment:
Naming thought: the "Attribute" suffix is not going to be very useful once it is literally a UsdAttribute. How about "NodeName" instead?

Reply:
great, i think this works well.